website/docs/docs/supported-data-platforms.md

```diff
@@ -1,85 +1,72 @@
 ---
 title: "Supported data platforms"
 id: "supported-data-platforms"
+sidebar_label: "Supported data platforms"
+description: "Connect dbt to any data platform in dbt Cloud or dbt Core, using a dedicated adapter plugin"
+hide_table_of_contents: true
 ---
 
-dbt connects to and runs SQL against your database, warehouse, lake, or query engine. We group all of these SQL-speaking things into one bucket called _data platforms_. dbt can be extended to any data platform using a dedicated _adapter plugin_. These plugins are built as Python modules that dbt Core discovers if they are installed on your system. All the adapters listed below are open source and free to use, just like dbt Core.
+dbt connects to and runs SQL against your database, warehouse, lake, or query engine. These SQL-speaking platforms are collectively referred to as _data platforms_. dbt connects with data platforms by using a dedicated [adapter plugin](/guides/dbt-ecosystem/adapter-development/1-what-are-adapters) for each. Plugins are built as Python modules that dbt Core discovers if they are installed on your system. 
```

Comment:
do we want to use 'data platforms' or 'adapters'? i understand the difference between them both (plugin versus platforms) but i'm not sure what the main message and term is for this page. is it adapters? or is it platforms?